src/components/ContactsList/ContactsEmptyList.jsx

```diff
@@ -29,21 +35,43 @@ export default class ContactsEmptyList extends React.Component {
 
     return (
       <Empty
+        className="contacts-empty"
         icon={EmptyIcon}
         title={t("empty.title")}
         text={hasConnector ? t("empty.after") : ""}
       >
         {!hasConnector && (
-          <IntentOpener
-            action="CREATE"
-            doctype="io.cozy.accounts"
-            options={{ slug: "google" }}
-            onComplete={this.afterConnection}
-          >
-            <Button label={t("empty.google")} />
-          </IntentOpener>
+          <div className="contacts-empty-actions-wrapper">
+            <span className="contacts-empty-action">
+              <IntentOpener
+                action="CREATE"
+                doctype="io.cozy.accounts"
+                options={{ slug: "google" }}
+                onComplete={this.afterConnection}
+              >
+                <Button
```

Comment:
Do we still want this button if there is now a modal that will manage contacts imports?

Reply:
The button is still here on the mockups, I could never see what it does since it doesn't work locally (I assumed my dev env was missing something).

@jsilvestre or @Claiw, any idea?